src/Components/Books.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import Navbar from './Navbar';
+
+function Books() {
+  return (
+    <div>
+      <Navbar />
+      <div className="container">
+        <h2>List of availabe books!</h2>
+      </div>
+
+    </div>
```

Comment:
Kindly you need to add a component for adding a new book as mentioned in the project requirements , with no functionality needed at this point :) :)